gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/io/gravitee/apim/core/plan/use_case/CreatePlanUseCaseTest.java

```diff
@@ -235,27 +235,29 @@ void should_create_push_plan_with_null_security_type() {
     }
 
     @Test
-    void should_throw_exception_when_creating_native_plan() {
+    void should_create_native_plan() {
```

Comment:
What do you think to add others test like ? : 
- should create native plan with flow
- cannot create native plan with more than one flow (📝 this is a specific rules for native kafka api. We have the same limitation for api.flows)